docs/serverless/README.md

```diff
@@ -0,0 +1,53 @@
+# Serverless Agent
+
+The Serverless Agent is a build of the Datadog Agent capable of running
+in serverless environments, to collect custom metrics, function's logs and soon
+function's traces. Today, its main usage is to build the Datadog Lambda
+Extension for AWS Lambda.
+
+## Concepts
+
+Today, the Serverless Agent is intended to run as a side-car of the serverless function.
+
+### Validate the presence of the Serverless Agent and order a flush
+
+The Serverless Agent is running an HTTP server on port 8124.
+
+With the Datadog Lambda Extension, this HTTP servers is listening for two
+different routes:
+    - /lambda/hello
+    - /lambda/flush
+
+These routes are not doing anything in specialized for AWS, however, they can be
```

Comment:
`These routes are not doing anything in specialized for AWS`
What does it mean?